python/hail/dataset.py

```diff
@@ -2336,6 +2336,16 @@ def mendel_errors(self, output, fam):
         pargs = ['mendelerrors', '-o', output, '-f', fam]
         self.hc._run_command(self, pargs)
 
+    def min_rep(self):
+        """
+        Gives minimal, left-aligned representation of alleles.
+
+        Example:
+            If a variant has REF = "TAA" and ALTs = ["TA","AA"], it will be modified to REF = "TA", ALTs = ["T","A"]
```

Comment:
can we give an example where the position changes?  Also note that this is possible in the doc.